<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fight in square - serwer</string>
  </property>
  <widget class="QWidget" name="serverCentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="serverLayout">
      <item>
       <layout class="QHBoxLayout" name="configLayout">
        <item>
         <widget class="QLabel" name="portLabel">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portEdit">
          <property name="text">
           <string>8001</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="onOffButton">
          <property name="text">
           <string>Uruchom serwer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectMapButton">
          <property name="text">
           <string>Wybór mapy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="gamersLabel">
        <property name="text">
         <string>Gracze:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="gamersList">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Drużyna</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Imię</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="logsLabel">
        <property name="text">
         <string>Logi:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="messageBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
